macros/anaUltraThreshold.py

```diff
@@ -7,7 +7,9 @@
 from PanChannelMaps import *
 from gempython.utils.nesteddict import nesteddict as ndict
 
-parser = OptionParser()
+from qcoptions import parser
+
+#parser = OptionParser()
```

Comment:
Where is this seg fault? We should be able to detect which vfats are present based on the data file. I think it would be better to make the analysis smart enough to see this and not need a --vfatmask option.